packages/marshal/src/encodeToSmallcaps.js

```diff
@@ -399,7 +399,9 @@ export const makeDecodeFromSmallcaps = ({
               }
             }
           }
-          case '+':
+          case '+': {
+            return BigInt(encoding.slice(1));
```

Comment:
I'm a little sad we have to do a `.slice` here as that can't be free performance-wise. Let's consider reverting once we land the XS update